subcommands/keys/ca_create.go

```diff
@@ -97,43 +95,35 @@ func doCreateCA(cmd *cobra.Command, args []string) {
 	resp, err := api.FactoryCreateCA(factory)
 	subcommands.DieNotNil(err)
 
-	writeFile("ca-csr", resp.CaCsr, 0400)
-	writeFile("tls-csr", resp.TlsCsr, 0400)
+	writeFile(x509.OnlineCaCsrFile, resp.CaCsr, 0400)
+	writeFile(x509.TlsCsrFile, resp.TlsCsr, 0400)
 	writeFile("create_ca", *resp.CreateCaScript, 0700)
 	writeFile("create_device_ca", *resp.CreateDeviceCaScript, 0700)
 	writeFile("sign_ca_csr", *resp.SignCaScript, 0700)
 	writeFile("sign_tls_csr", *resp.SignTlsScript, 0700)
```

Comment:
If you defined a few constants here; I guess it's worth defining a constant for all these file names, right?

Reply:
I did not create constants for the scripts names. But since the same scripts names are used here and inside the bash implementation, it seems to be worth it. I'll do it.